doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
+           
+           # Initialize the Operator class by calling its __init__ method.
+           # This sets properties such as domain and range and allows the other
+           # operator convenience functions to work.
+           odl.Operator.__init__(self, domain=kernel.space, range=kernel.space,
+                                 linear=True)
+   
+       def _call(self, x):
+           """Implement calling the operator by calling scipy."""
+           return scipy.signal.convolve(self.kernel, x, mode='same')
+   
+We can verify that our operator works by calling it on some data. This can either come from some outside source, or from simulations. ODL also provides a nice range of standard phantoms such as the `cuboid` and `shepp_logan` phantoms:
+
+.. code-block:: python
+
+   space = odl.uniform_discr([-1, -1], [1, 1], [100, 100])
+   kernel = odl.phantom.cuboid(space, [-0.05, -0.05], [0.05, 0.05])
+   conv = Convolution(kernel)
+   phantom = odl.phantom.shepp_logan(space, modified=True)
+   data = conv(phantom)
+   
+   kernel.show('kernel')
+   phantom.show('phantom')
+   data.show('convolved phantom')
+
+.. image:: figures/getting_started_kernel.png
+
+.. image:: figures/getting_started_phantom.png
+
+.. image:: figures/getting_started_convolved.png
+
+We can use this data to try to solve the inverse problem. To solve that, we need further properties of the operator. Specifically, we need an `Operator.adjoint`. Luckily, the convolution operator is self adjoint if the kernel is symmetric, so we can add: 
+
+.. code-block:: python
+
+   class Convolution(odl.Operator):
+       ...  # old code
+
+       @property  # making the adjoint a property lets users access it as conv.adjoint
+       def adjoint(self):
+           return self  # the adjoint is the same as the operator
+   
+Now lets try with the simple `landweber` solver:
+
+.. code-block:: python
+
+   # Need operator norm for step length (omega)
+   opnorm = odl.power_method_opnorm(conv, niter=10)
+   
+   x = space.zero()
+   odl.solvers.landweber(conv, x, data, niter=100, omega=1/opnorm**2)
+   x.show('landweber solution')
+   
+.. image:: figures/getting_started_landweber.png
+
+This solution is not very good, mostly due to the ill-posedness of the convolution operator. 
+Other solvers, like the `conjugate_gradient` solver give similar results:
+
+.. code-block:: python
+
+   x = space.zero()
+   odl.solvers.conjugate_gradient(conv, x, data, niter=100)
+   x.show('conjugate gradient solution')
+
+.. image:: figures/getting_started_conjugate_gradient.png
+
+A method to remedy this problem is to instead consider a regularized problem. One of the classic regularizers is `Tikhonov regularization <https://en.wikipedia.org/wiki/Tikhonov_regularization>`_ where we instead study the problem
+
+.. math::
+   
+   \min_f ||Af - g||_2 + ||Bf||_2
+
+Where :math:`B` is a regularizer. To solve this with the above solvers, we can find the first order optimality conditions
```

Comment:
"where" not capitalized
"regularizer" is not really precise, rather "'roughening' operator". Basically one wants that Bf is less smooth than f so one gets something more smooth in the other direction.

"solve this" -> "solve it"